.github/workflows/performance-tests.yml

```diff
@@ -0,0 +1,195 @@
+name: Performance tests
+on:
+  pull_request:
+    branches:
+      - master
+    types:
+      - labeled
+      - synchronize
+jobs:
+  run-k6-performance-tests:
+    name: Run k6 performance tests
+    runs-on: ubuntu-latest
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'ci-performance-tests') }}
+    env:
+      USE_GKE_GCLOUD_AUTH_PLUGIN: "True"
+      INFRA_TOKEN: ${{ secrets.INFRA_TOKEN }}
+      KUBECONFIG: artifacts/kubeconfig
+      TERM: xterm
+      DEFAULT_IMAGE_REGISTRY: "quay.io/rhacs-eng"
+      SCALE_CLUSTER_NAME: k6-test-${{ github.event.pull_request.number }}
+      SCALE_WORKLOAD_TYPE: default
+      SCALE_WORKLOAD_CLUSTERS: 10
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Checkout workflow scripts
+        uses: actions/checkout@v4
+        with:
+          repository: "stackrox/workflow"
+          path: workflow
+
+      # # Create cluster
+      # - uses: stackrox/actions/infra/create-cluster@main
+      #   with:
+      #     token: ${{ secrets.INFRA_TOKEN }}
+      #     flavor: demo
+      #     name: k6-test
+      #     lifespan: 24h
+      #     wait: "true"
+
+      # Create scale cluster with larger nodes.
+      - name: Create testing cluster
+        uses: stackrox/actions/infra/create-cluster@main
+        with:
+          token: ${{ secrets.INFRA_TOKEN }}
+          flavor: gke-default
+          name: ${{ env.SCALE_CLUSTER_NAME }}
+          lifespan: 12h
+          args: nodes=3,machine-type=e2-standard-32
+          wait: true
+
+      # Fetch infractl artifacts
+      - uses: stackrox/actions/infra/install-infractl@v1
+      - uses: "google-github-actions/auth@v1"
+        with:
+          credentials_json: "${{ secrets.GCP_PERFORMANCE_TESTER_CI_SA }}"
+      - name: "Set up Cloud SDK"
+        uses: "google-github-actions/setup-gcloud@v1"
+        with:
+          install_components: "gke-gcloud-auth-plugin"
+      - name: Test cluster readiness
+        run: |
+          STATUS=$(infractl get "${SCALE_CLUSTER_NAME//./-}" --json | jq -r .Status)
+          if [ "$STATUS" != "READY" ]; then
+            exit 1
+          fi
+      - name: Fetch infra artifacts
+        run: |
+          infractl artifacts "${SCALE_CLUSTER_NAME//./-}" --download-dir artifacts > /dev/null
+
+      - name: Add Docker login for Quay.io
+        env:
+          REGISTRY_USERNAME: ${{ secrets.QUAY_RHACS_ENG_RO_USERNAME }}
+          REGISTRY_PASSWORD: ${{ secrets.QUAY_RHACS_ENG_RO_PASSWORD }}
+        run: |
+          docker login -u "${REGISTRY_USERNAME}" --password-stdin quay.io <<<"${REGISTRY_PASSWORD}"
+
+      - name: Clean previous installation
+        run: |
+          source "${{ github.workspace }}"/workflow/env.sh
```

Comment:
Why did you add `${{ github.workspace }}`?
That should not be required, as the runner executes in that directory by default.